pinot-core/src/main/java/com/linkedin/pinot/core/io/readerwriter/RealtimeIndexOffHeapMemoryManager.java

```diff
@@ -84,6 +103,7 @@ public PinotDataBuffer allocate(long size, String columnName) {
   public void close() throws IOException {
     for (PinotDataBuffer buffer : _buffers) {
       buffer.close();
+      _serverMetrics.addValueToTableGauge(_tableName, ServerGauge.REALTIME_OFFHEAP_MEMORY_USED, -buffer.size());
```

Comment:
we should report full value (totalMemBytes - buffer.size() ? )

Reply:
We are subtracting he buffer size here, but yes, alternatively we can subtract totalMemBytes from the guage.